catalog/couchdb/couchdb-cluster.bom

```diff
@@ -35,12 +35,22 @@ brooklyn.catalog:
       services:
       - type: org.apache.brooklyn.entity.nosql.couchdb.CouchDBNode
         brooklyn.config:
+          provisioning.properties:
+            customizer:
+              $brooklyn:object:
+                type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+                object.fields:
+                  enabled: $brooklyn:config("couchdb.sharedsecuritygroup.create")
+                  tcpPortRanges:
+                    - "1024-65535"
```

Comment:
I think it would always be best practice to run this cluster behind a firewall, these ports are open between the machines rather than to the public. I agree limiting it to a range would be better and that would be a good next step.